### ISSUE TITLE
[Bug]: Multipart error dialog cannot be closed

### ISSUE THREAD

**Issue by EvanVujcec** [httpie/desktop#31]
### What happened?

When attaching a binary file to a form that requires multipart/* and then attempting to toggle off the multipart, the app remains stuck on the dialog box that appears saying "Form contains binary files so can't be converted to urlencoded". There is no click-away area around the dialog and no close button. Initially discovered on Windows but macOS and web app exhibit the same behavior. Refresh works which does "close" the dialog but this isn't obvious.

### Steps to reproduce?

```Markdown
1. Begin request
2. Select Form for Body
3. Add file to form value (click change in the multipart change dialog if necessary)
4. Click the multipart toggle in the bottom left
```

### What did you expect to happen?

Dialog to close

### Platform

Windows

### HTTPie Desktop version

2022.11.1

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by jakubroztocil** [comment]
Thanks for the report @EvanVujcec! We’re looking into it.

**Comment by jakubroztocil** [comment]
@EvanVujcec it’s fixed in [v2022.12.0](https://github.com/httpie/desktop/releases/tag/v2022.12.0) 🙌
